acceptance-tests/profiler-stress/runner.cs

```diff
@@ -124,6 +124,11 @@ static int Main ()
 			if (Environment.GetEnvironmentVariable ("MONO_PROFILER_STRESS_OPTIONS") is string envOptions)
 				options = envOptions;
 
+			var suspend = false;
+
+			if (Environment.GetEnvironmentVariable ("MONO_PROFILER_STRESS_SUSPEND") is string envSuspend)
```

Comment:
If we don't want to suspend wouldn't we just not pass the env variable? The usual check we have is `Environment.GetEnvironmentVariable ("MONO_PROFILER_STRESS_SUSPEND") != null`, thus avoiding all king of possible exception in `bool.Parse`.

Reply:
That could be misleading too, since you'd expect `MONO_PROFILER_STRESS_DEBUG=0` (or something similar) to disable it, but it wouldn't.

Personally I don't think it matters too much either way since this isn't something user-facing.